egps/egp-5/egp-5.ts

```diff
@@ -39,21 +41,35 @@ async function proposal() {
   // --- main egp logic ---
   /**
    * Steps:
-   * 1. Transfer `amountPastTerms` worth of ELFI to the `aidrop` contract.
-   * 2. Transfer `amountFutureTerms` worth of ELFI to the `futureRewards` contract.
+   * 1. Deploy airdrop contract
+   * 2. Transfer `amountPastTerms` worth of ELFI to the `aidrop` contract.
+   * 3. Transfer `amountFutureTerms` worth of ELFI to the `futureRewards` contract.
    */
 
+  // Deploy the airdrop contract
+  const airdropDeployer = new ethers.ContractFactory(
+    airdropData.abi,
+    airdropData.bytecode,
+    signer
+  );
+  const airdropContract = await airdropDeployer.deploy(
+    signer,
```

Comment:
it looks like you're setting the [gov address for reclaim after expiration](https://github.com/element-fi/council/blob/main/contracts/features/Airdrop.sol#L20) to your personal wallet.... this should probably be CoreVoting or the Timelock.

Reply:
Left that in there for you to catch 🙄